mlflow/protos/databricks_uc_registry_messages.proto

```diff
@@ -190,6 +192,26 @@ enum StorageMode {
   DEFAULT_STORAGE = 2;
 }
 
+message EncryptionDetails {
+  oneof encryption_details_type {
+    // Details for CLOUD_MANAGED_SSE_KEYS:
+    SseEncryptionDetails sse_encryption_details = 1;
+  }
+}
+
+enum SseEncryptionAlgorithm {
+  SSE_ENCRYPTION_ALGORITHM_UNSPECIFIED = 0;
+  AWS_SSE_KMS = 2;   // "aws:kms" in x-amz-server-side-encryption' header
```

Comment:
```suggestion
  AWS_SSE_KMS = 1;   // "aws:kms" in x-amz-server-side-encryption' header
```
Unless there's a reason why this needs to start at 2?

Reply:
This is to match the UCMR proto, derived from the UC proto. We're only exposing the information we need.